composer.json

```diff
@@ -26,9 +27,11 @@
 		"phpunit-unit": "phpunit --testsuite unit",
 		"phpunit-integration": "phpunit --testsuite integration",
 		"phpunit-coverage": "phpunit --coverage-clover=coverage.clover",
+		"phpcs": "phpcs -ps",
 		"test": [
 			"@lint",
-			"@phpunit"
+			"@phpunit",
```

Comment:
Good point, then it will match the order that Travis is doing it too. And fail quicker.